superx/routes/home.py

```diff
@@ -135,4 +134,21 @@ def remove_item():
 
     session['cart'] = cart_list
 
+
+    return ''
+
+
+def city():
+    """
+    adds city that was chosen, using jquery to get the data and ajax so not to redirect
+    """
+    session['city'] = request.form.get('city')
```

Comment:
in pr #71  @OdedHellman and I discussed this functionality and moved it directly into routing.py.
My suggestion is to keep it that way.